### ISSUE TITLE
[T9][Accessibility][Functional]: unable to make selection in familly tab - Quicklist

### ISSUE THREAD

**Issue by SR027213** [SR027213/Feb-19-Bug-Bash#30]
Steps to Reproduce:
Unable to make selection in familly tab - Quicklist
